conftest.py

```diff
@@ -19,12 +20,8 @@
 import sct_download_data as downloader
 
 
-@pytest.fixture(scope='session', autouse=True)
-def download_data(request):
-    # This is a hack because the capsys fixture can't be used with
-    # session scope at the moment.
-    # https://github.com/pytest-dev/pytest/issues/2704#issuecomment-603387680
-    capmanager = request.config.pluginmanager.getplugin("capturemanager")
-    with capmanager.global_and_fixture_disabled():
+def pytest_collectstart():
+    # TODO [AJ] check integrity eventually
+    if not os.path.exists(sct_test_path()):
         print('\nDownloading sct testing data.')
-        downloader.main(['-d', 'sct_testing_data', '-o', sct_test_path()])
+        downloader.main(['-d', 'sct_testing_data', '-o', sct_test_path()])
```

Comment:
There is overlap with #2903, right? i.e. if that PR gets merged first, these changes won't be needed?

Reply:
Yes sorry for the confusion, I will rebase cleanly.

Reply:
No worries! Thanks much. :smile: